File: core/src/test/java/utils/RandomIntegerTest.java
package utils;

import net.rickiekarp.core.util.CommonUtil;
import org.junit.Test;

import static org.junit.Assert.assertTrue;

public class RandomIntegerTest {

    @Test
    public void testNumberGeneration() {
        int actual;
        for (int i = 1; i <= 10; i++) {
            actual = CommonUtil.randInt(1, i);
            assertTrue(actual >= 1 && actual <= i);
        }
    }
}

File: projects/JavaFXApps/core/src/test/java/utils/RandomIntegerTest.kt
package utils

import net.rickiekarp.core.util.CommonUtil
import org.junit.jupiter.api.Test

import org.junit.jupiter.api.Assertions.assertTrue

internal class RandomIntegerTest {

    @Test
    fun testNumberGeneration() {
        var actual: Int
        for (i in 1..10) {
            actual = CommonUtil.randInt(1, i)
            assertTrue(actual >= 1 && actual <= i)
        }
    }
}
